lib/ansible/module_utils/ec2.py

```diff
@@ -168,8 +168,9 @@ def get_aws_connection_info(module, boto3=False):
         elif 'EC2_ACCESS_KEY' in os.environ:
             access_key = os.environ['EC2_ACCESS_KEY']
         else:
```

Comment:
Can you match the elif-y style here as:

```
elif boto.config.get('Credentials', 'aws_access_key_id'):
    # set it
elif boto.config.get('default', 'aws_access_key_id'):
    # set it
else:
    access_key = None
```
Functionally it's the same, but since we've got this big long elif already, it's preferable to match it.